corehq/apps/app_manager/views/apps.py

```diff
@@ -678,7 +679,8 @@ def should_edit(attribute):
 
     if should_edit("name"):
         clear_app_cache(request, domain)
-        name = hq_settings['name']
+        name = html.strip_tags(hq_settings['name'])
+        app.name = name
```

Comment:
This is wrong, `app.name` is already getting set above on line 677/8. I also don't think you should use `strip_tags`, if we're going to sanitize this at all on the server side (seems like there's some feeling that we shouldn't) then we should use `html.escape` instead.

The same thing goes for the other places that use `html.strip_tags`